src/XrdTls/XrdTlsTempCA.cc

```diff
@@ -50,8 +50,9 @@
 #include <atomic>
 
 namespace {
-    
-typedef std::unique_ptr<FILE, decltype(&fclose)> file_smart_ptr;
+
+static void FClose(FILE *f) {fclose(f);}
```

Comment:
Is adding this really necessary? I wonder if just adjusting the typedef to use `void(*)(FILE*)` instead of `decltype(&fclose)` is already enough.

Reply:
Yes, that works too. I have changed the PR accordingly.